helm-charts/splunk-otel-collector/templates/_helpers.tpl

```diff
@@ -441,6 +441,26 @@ Whether clusterReceiver should be enabled
 {{- end -}}
 
 
+{{/*
+Whether persistentQueue should be enabled
+*/}}
+{{- define "splunk-otel-collector.persistentQueueEnabledLogs" -}}
+{{ $gatewayDisabled := ne (include "splunk-otel-collector.gatewayEnabled" .) "true" }}
+{{- and $gatewayDisabled (.Values.splunkPlatform.sendingQueue.persistentQueueEnabled.logs) (and (ne (include "splunk-otel-collector.distribution" .) "eks/fargate") (ne (include "splunk-otel-collector.distribution" .) "gke/autopilot")) -}}
+{{- end -}}
+
+
+{{- define "splunk-otel-collector.persistentQueueEnabledMetrics" -}}
+{{ $gatewayDisabled := ne (include "splunk-otel-collector.gatewayEnabled" .) "true" }}
+{{- and $gatewayDisabled (.Values.splunkPlatform.sendingQueue.persistentQueueEnabled.metrics) (and (ne (include "splunk-otel-collector.distribution" .) "eks/fargate") (ne (include "splunk-otel-collector.distribution" .) "gke/autopilot")) -}}
+{{- end -}}
+
+
+{{- define "splunk-otel-collector.persistentQueueEnabled" -}}
```

Comment:
I'm a bit fuzzy on why you have this boolean and how you use it. I'll comment below on it.

Reply:
yeah, got rid of it as it served little purpose. It was used only once, in clusterReciever's deployment.